x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_item_card/exception_item_card_conditions.tsx

```diff
@@ -0,0 +1,128 @@
+  [ListOperatorTypeEnum.MATCH_ANY]: i18n.CONDITION_OPERATOR_TYPE_NOT_MATCH_ANY,
+  [ListOperatorTypeEnum.MATCH]: i18n.CONDITION_OPERATOR_TYPE_NOT_MATCH,
+});
+
+const EuiFlexGroupNested = styled(EuiFlexGroup)`
+  margin-left: ${({ theme }) => theme.eui.euiSizeXL};
+`;
+
+const EuiFlexItemNested = styled(EuiFlexItem)`
+  margin-bottom: 6px !important;
+  margin-top: 6px !important;
+`;
+
+export interface CriteriaConditionsProps {
+  entries: ExceptionListItemSchema['entries'];
+  dataTestSubj: string;
+}
+
+export const ExceptionItemCardConditions = memo<CriteriaConditionsProps>(
+  ({ entries, dataTestSubj }) => {
+    const getEntryValue = (type: string, value: string | string[] | undefined) => {
+      if (type === 'match_any' && Array.isArray(value)) {
+        return value.map((currentValue) => <EuiBadge color="hollow">{currentValue}</EuiBadge>);
+      }
+      return value ?? '';
+    };
+
+    const getEntryOperator = (type: string, operator: string) => {
+      if (type === 'nested') return '';
+      return operator === 'included'
+        ? OPERATOR_TYPE_LABELS_INCLUDED[type as keyof typeof OPERATOR_TYPE_LABELS_INCLUDED] ?? type
+        : OPERATOR_TYPE_LABELS_EXCLUDED[type as keyof typeof OPERATOR_TYPE_LABELS_EXCLUDED] ?? type;
+    };
+
+    const getNestedEntriesContent = useCallback(
+      (type: string, nestedEntries: NonEmptyNestedEntriesArray) => {
+        if (type === 'nested' && nestedEntries.length) {
+          return nestedEntries.map((entry) => {
+            const { field: nestedField, type: nestedType, operator: nestedOperator } = entry;
+            const nestedValue = 'value' in entry ? entry.value : '';
+
+            return (
+              <EuiFlexGroupNested
+                data-test-subj={`${dataTestSubj}-nestedCondition`}
+                key={nestedField + nestedType + nestedValue}
+                direction="row"
+                alignItems="center"
+                gutterSize="m"
+                responsive={false}
+              >
+                <EuiFlexItemNested grow={false}>
+                  <EuiToken iconType="tokenNested" size="s" />
+                </EuiFlexItemNested>
+                <EuiFlexItemNested grow={false}>
+                  <EuiExpression description={''} value={nestedField} color="subdued" />
+                </EuiFlexItemNested>
+                <EuiFlexItemNested grow={false}>
+                  <EuiExpression
+                    description={getEntryOperator(nestedType, nestedOperator)}
+                    value={getEntryValue(nestedType, nestedValue)}
+                  />
+                </EuiFlexItemNested>
+              </EuiFlexGroupNested>
+            );
+          });
+        }
+      },
+      [dataTestSubj]
+    );
+
+    return (
+      <div data-test-subj={dataTestSubj}>
+        {entries.map((entry, index) => {
+          const { field, type } = entry;
+          const value = 'value' in entry ? entry.value : '';
```

Comment:
Just out of curiosity, do we have here any specific reason to use the `in` operator and not something like:
`const value = entry?.value ?? '' `

Reply:
It's funky because of the typescript union in which a specific property (in this case `value`) is not present in all union types, so `?` doesn't solve it in this case. I did try, but typescript still yelled at me.

Reply:
Sure, you're right!

Found this [issue](https://github.com/microsoft/TypeScript/issues/33736)